src/helpers/map-helper.js

```diff
@@ -98,6 +98,7 @@ const selectSensor = (id, map, geocoder) => {
   const paintProperty = getPaintProperty(id);
   const sensor = sensors.get(id);
   store.commit("cons/setSensor", { sensor });
+  store.commit("timelapse/displayTimelapse");
```

Comment:
I am not sure if we discussed the intended behavior, but do we want the time-lapse components to disappear when a sensor is un-selected and the user is not on the inundation layer?